Source/Diagnostics/Diagnostics.H

```diff
@@ -266,6 +270,12 @@ protected:
      *  The second vector is loops over the total number of levels.
      */
     amrex::Vector< amrex::Vector< amrex::MultiFab > >  m_mf_output;
+    /** summation multifab, where all fields (computed, cell-centered, and stacked)
+     *  are summed for every step in a time averaging period.
+     *  The first vector is for total number of snapshots. (=1 for FullDiagnostics)
```

Comment:
Just checking, `=1` in this comment means something like "only one element"? If so (that is, if `1` does not refer to the index to be used to access that vector layer), a more verbose description might be clearer.

Reply:
Unsure. I think @RevathiJambunathan may have added/suggested this comment.

Reply:
Actually, that was copy-pasted and then modified from the comment of the `m_mf_output` above.
We could consider making this in all the places where it appears in this file.
But a separate PR would probably be better for this.